### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by kevin-10**
- Fixed Tinker bods
- Added Energy pick for crimson/plat drakes
- Removed conflicting layer from Merchant Crew
- Fixed spells that cannot stack damage
- Fixed Stack Overflow
- Failed Pet Special Abilities and Area Effects no longer use mana
- Void Pool Context Menu selection is only available if you've visited teh void pool

### CODE REVIEW

File: Scripts/Services/BulkOrders/SmallBODs/SmallTinkerBOD.cs [hunk 1]
```diff
@@ -191,6 +195,8 @@ public override bool CheckType(Type itemType)
             new Type[] { typeof(Clock), typeof(ClockRight), typeof(ClockLeft) },
             new Type[] { typeof(GoldRing), typeof(SilverRing) },
             new Type[] { typeof(GoldBracelet), typeof(SilverBracelet) },
+            new Type[] { typeof(GoldBracelet), typeof(SilverBracelet) },
```
Comment (Argalep) [inline Scripts/Services/BulkOrders/SmallBODs/SmallTinkerBOD.cs]: duplicate ?

Comment (kevin-10) [reply Scripts/Services/BulkOrders/SmallBODs/SmallTinkerBOD.cs]: Good catch.

**Comment by kevin-10** [comment]
No, a slight change in teh method where the creature chooses a dispel spell.

**Comment by Tasanar** [comment]
So it could potentially still crash

**Pull request merged**